test/DynamoCoreTests/Logging/DynamoLoggerTest.cs

```diff
@@ -31,7 +31,7 @@ internal DynamoLogger GetDynamoLoggerWithTestModeFalse()
                 VerboseLogging = true
             };
 
-            var logger = new DynamoLogger(debugSettings, pathManager.LogDirectory, false);
+            var logger = new DynamoLogger(debugSettings, pathManager.LogDirectory, false, false, false);
```

Comment:
Does it make sense to make some args optional in this method?

Reply:
We have a task files for cleaning this up: DYN-5757. I think we are good for now.